src/components/Asset/AssetActions/Download/ContractingProvider/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactElement, useCallback, useEffect, useState } from 'react'
+import styles from './index.module.css'
+import Button from '../../../../@shared/atoms/Button'
+import { useAccount, useSignMessage } from 'wagmi'
+import { getContractingProviderNonce, getPayPerUseCount } from '../../utils'
+import Alert from '../../../../@shared/atoms/Alert'
+import { useMarketMetadata } from '../../../../../@context/MarketMetadata'
+
+export enum PAYMENT_MODES {
+  SUBSCRIPTION = 'subscription',
+  PAYPERUSE = 'payperuse'
+}
+
+export type PaymentMode = `${PAYMENT_MODES}`
```

Comment:
question: Why putting `PAYMENT_MODES` into a string?
```suggestion
export type PaymentMode = PAYMENT_MODES
```